iib/web/migrations/versions/04dd7532d9c5_polymorphic_requests.py

```diff
@@ -0,0 +1,397 @@
+"""
+Use polymorphic tables for different types of build requests.
+
+Revision ID: 04dd7532d9c5
+Revises: 274ba38408e8
+Create Date: 2020-04-17 14:52:40.766500
+
+"""
+from alembic import op
+from sqlalchemy.sql.expression import select
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '04dd7532d9c5'
+down_revision = '274ba38408e8'
+branch_labels = None
+depends_on = None
+
+
```

Comment:
Should a `REQUEST_TYPE_GENERIC = 0` be added here?

Reply:
It's not needed because we never migrate generic request types, which don't really exist.